src/commands/inputs.rs

```diff
@@ -0,0 +1,167 @@
+use anyhow::{Context, Result};
+use clap::Parser;
+use indexmap::IndexMap;
+use serde_json::{Value, json};
+use std::path::{Path, PathBuf};
+use url::Url;
+use wdl::{
+    analysis::{path_to_uri, types::Type},
+    ast::{
+        AstToken, Document, SyntaxKind, Visitor,
+        v1::{
+            self,
+            Expr::{self, Literal},
+        },
+    },
+    cli::analyze,
+    doc,
+    grammar::SyntaxTree,
+};
+
+#[derive(Parser, Debug)]
+#[command(about = "Generate input JSON from a WDL document", version, about)]
+pub struct InputsArgs {
+    #[arg(required = true)]
+    #[clap(value_name = "input path")]
+    pub document: String,
+
+    #[arg(short, long)]
+    #[clap(value_name = "workflow name")]
+    pub name: Option<String>,
+
+    #[arg(short, long)]
+    #[clap(value_name = "output path")]
+    pub output: Option<PathBuf>,
+
+    #[arg(short, long)]
+    #[clap(value_name = "literal defaults")]
+    pub literal_defaults: bool,
+
+    #[arg(short, long)]
+    #[clap(value_name = "override expressions")]
+    pub override_expressions: bool,
```

Comment:
Sorry if our thread got confusing, I did at one point suggest these two flags, but me and Andrew T at the end of the discussion decided the behavior flags should be `--hide-expressions`, `--hide-defaults`, and `--nested-inputs`. See the comment thread for what these flags should do.